README.md

```diff
@@ -143,6 +143,24 @@ If you're using Laravel and want to support **realtime broadcasting and events**
 
 > If you want **ably-php** as a rest dependency across service providers, check [ably-php-laravel](https://packagist.org/packages/ably/ably-php-laravel). **ably-php-laravel** is a simple wrapper over **ably-php** with laravel-specific classes. This has limited use-cases and **laravel-broadcaster** is recommended over **ably-php-laravel** for most use-cases.
 
+### Making explicit HTTP requests to Ably Rest Endpoints / Batch publish
+- The `AblyRest->Request` method can be used to make explicit HTTP requests to the [Ably REST API](https://ably.com/docs/api/rest-api).
+- It automatically adds necessary auth headers based on the initial auth config and supports pagination.
+- The following is an example of using the batch publish API based on the [Ably batch publish rest endpoint documentation](https://ably.com/docs/api/rest-api#batch-publish).
+
+```php
+    $payload = array(
+        "channels" => ["channel1", "channel2", "channel3", "channel4"],
+        "messages" => array(
+            "id" => "1",
+            "data" => "foo"
+        )
+    );
+    $batchPublishPaginatedResult = $client->request("POST", "/messages", [], $payload);
```

Comment:
We should probably also recommend explicitly that the payload is not serialised to JSON and that an `array` be passed in (and we'll handle the serialisation). That would at least give explicit instructions to help avoid the issue we saw before.

Reply:
Added note https://github.com/ably/ably-php/pull/198/commits/be405704415943c7773aa767bbad2cda5d513d26